app/javascript/components/HomePageContent.js

```diff
@@ -1,15 +1,8 @@
 import React from 'react'
 
 import HomePageContentLegacy from './HomePageContentLegacy'
-import HomePageContentXds from './HomePageContentXds'
-import { getFeatureFlagsWithDefaults } from 'providers/UserProvider'
 
 /** Until refactor is done, this merely sets appropriate component */
 export default function HomePageContent() {
-  const flags = getFeatureFlagsWithDefaults()
-  if (flags.cross_dataset_search_frontend) {
-    return (<HomePageContentXds />)
-  } else {
-    return (<HomePageContentLegacy />)
-  }
+  return (<HomePageContentLegacy />)
```

Comment:
It'd help to move inline`HomePageContentLegacy.js` into this `HomePageContent.js`, or at least ticket as much for next sprint prep and note it as a ticket (e.g. "TODO (SCP-4321): inline HomePageContentLegacy.js into module").